src/shared_utilities/drivers.jl

```diff
@@ -1102,7 +1114,7 @@ function prescribed_forcing_era5(
         regridder_type,
         file_reader_kwargs = (; preprocess_func = (data) -> -data / 1000,),
         method = time_interpolation_method,
-        compose_function = (mtpr, msr) -> mtpr - msr,
+        compose_function = (mtpr, msr) -> min.(mtpr .- msr, Float32(0)),
```

Comment:
I noticed that even if  mtpr - msr > 0, when regridded, this may not hold. This enforced that precipitation has the correct sign